movie_REST_API/movie_REST_API/Controllers/MovieViewController.swift

```diff
@@ -0,0 +1,247 @@
+        static let baseImageName = "фон4"
+        static let baseImageFilmName = "film"
+        static let newFilmString = "Новинки"
+    }
+    
+    // MARK: - Private Visual Components
+    private let newButton: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = .systemGreen
+        button.setTitle(Constant.newFilmString, for: .normal)
+        button.setTitleColor(UIColor.black, for: .normal)
+        button.titleLabel?.font = .boldSystemFont(ofSize: 20)
+        button.layer.cornerRadius = 15
+        button.clipsToBounds = true
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+    
+    private let baseImageView: UIImageView = {
+        let image = UIImageView()
+        image.contentMode = .scaleAspectFit
+        image.image = UIImage(named: Constant.baseImageFilmName)
+        image.layer.cornerRadius = 15
+        image.clipsToBounds = true
+        image.contentMode = .scaleAspectFill
+        image.translatesAutoresizingMaskIntoConstraints = false
+        return image
+    }()
+
+    private let popularButton: UIButton = {
+        let button = UIButton()
+        button.layer.cornerRadius = 10
+        button.layer.shadowColor = UIColor.black.cgColor
+        
+        button.layer.shadowRadius = 9.0
+        button.layer.shadowOpacity = 0.8
+        button.layer.shadowOffset = CGSize(width: 2.5, height: 2.5)
+
+        button.backgroundColor = .systemBlue
+        button.setTitle(Constant.allFilmString, for: .normal)
+        button.setTitleColor(UIColor.black, for: .normal)
+        button.tag = 0
+        button.translatesAutoresizingMaskIntoConstraints = false
+
+        return button
+    }()
+
+    private let rateButton: UIButton = {
+        let button = UIButton()
+        button.layer.cornerRadius = 10
+        button.layer.shadowColor = UIColor.black.cgColor
+        button.layer.shadowRadius = 9.0
+        button.layer.shadowOpacity = 0.8
+        button.layer.shadowOffset = CGSize(width: 2.5, height: 2.5)
+        button.backgroundColor = .systemCyan
+        button.setTitle(Constant.popularFilmString, for: .normal)
+        button.setTitleColor(UIColor.black, for: .normal)
+        button.tag = 1
+        button.translatesAutoresizingMaskIntoConstraints = false
+
+        return button
+    }()
+
+    private let tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.backgroundView = UIImageView(image: UIImage(named: Constant.baseImageName))
+        tableView.register(MovieViewCell.self, forCellReuseIdentifier: Constant.filmIdentifier)
+        tableView.showsVerticalScrollIndicator = false
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    // MARK: - Private Properties
+    private var isPress = true
+    private var movieViewModel = MovieViewModel()
```

Comment:
```suggestion
    private var movieViewl = MovieViewModel()
```
Model не пишем это слово ема е